docs/configuring-socorro.rst

```diff
@@ -27,13 +27,37 @@ The Socorro systemd service scripts use envconsul
 (https://github.com/hashicorp/envconsul) to read the configuration from Consul
 and set the environment.
 
-Below is the minimum viable configuration to get collection and
-processing working on a single node via Consul's REST interface::
+To keep track of your configuration, it's recommended to make a "socorro-config"
+directory and store these values there, and then load those into Consul as
+a separate step.
 
-    curl -s -X PUT -d "socorro.webapi.servers.WSGIServer" localhost:8500/v1/kv/socorro/collector/web_server.wsgi_server_class
-    curl -s -X PUT -d "/path/to/your/symbols" localhost:8500/v1/kv/socorro/processor/processor.raw_to_processed_transform.BreakpadStackwalkerRule.processor_symbols_pathname_list
+Below is the minimum viable configuration to get collection working on a
+single node:
 
-Note that Consul also has a Web UI you can use to get/set keys if you prefer.
+.. code-block:: bash
+
+    # Run collector in WSGI mode, instead of the default dev server
+    web_server__wsgi_server_class='socorro.webapi.servers.WSGIServer'
+
+Put this into a file named "collector.conf" in your socorro-config folder. 
+
+For processing to work, the path to your debug symbols must be provided:
+
+.. code-block:: bash
+
+    # This should be replaced with the path to your debug symbols.
+    processor__raw_to_processed_transform__BreakpadStackwalkerRule__processor_symbols_pathname_list='@@@PATH_TO_YOUR_SYMBOLS@@@'
```

Comment:
Note on the above, we did end up changing the default to `~/socorro/symbols` so I will make this change in the doc.